sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c

```diff
@@ -293,12 +293,36 @@ static void _send(gnrc_pktsnip_t *pkt)
         return;
     }
 
-#ifdef MODULE_GNRC_SIXLOWPAN_IPHC
-    if (netif->flags & GNRC_NETIF_FLAGS_6LO_HC) {
-        gnrc_sixlowpan_iphc_send(pkt, NULL, 0);
+    if (IS_USED(MODULE_GNRC_SIXLOWPAN_IPHC) &&
+        netif->flags & GNRC_NETIF_FLAGS_6LO_HC) {
+        gnrc_sixlowpan_frag_fb_t *fbuf;
+
+        if (IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_HINT) &&
+            IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_MINFWD)) {
+            /* prepare for sending with IPHC slack in first fragment */
```

Comment:
Can we already check if `!ipv6_addr_is_link_local()` here?

Reply:
Mhhh... would be pretty complicated, so I rather would keep this as a later optimization.